src/app/shared/components/course-info/course-info.component.html

```diff
@@ -9,6 +9,6 @@
   </div>
   <div class="info">
     <div class="info-type">Authors:</div>
-    <div class="info-content">{{authors}}</div>
+    <div class="info-content" *ngFor="let authorId of authors">{{authorId}}</div>
```

Comment:
Good practice: Use trackBy on ngFor. It helps angular to identify the items of the list when selecting/deleting records.

https://angular.io/api/core/TrackByFunction